driver-core/src/test/functional/com/mongodb/operation/AggregateToCollectionOperationSpecification.groovy

```diff
@@ -297,7 +300,8 @@ class AggregateToCollectionOperationSpecification extends OperationFunctionalSpe
         def hint = new BsonDocument('a', new BsonInt32(1))
         collectionHelper.createIndex(hint)
 
-        def operation = new AggregateToCollectionOperation(getNamespace(), [Aggregates.out('outputCollection')])
+        def operation = new AggregateToCollectionOperation(getNamespace(),
+                [Aggregates.out('outputCollection').toBsonDocument(BsonDocument, registry)])
```

Comment:
Dynamic languages... Have to call Bson#toBsonDocument because Aggregates#out no longer returns a BsonDocument.  It now returns a com.mongodb.client.model.Aggregates.OutStage